backend/config/curation-api.yml

```diff
@@ -46,32 +53,34 @@ paths:
 
   /v1/collections:
     get:
-      summary: Fetch Collections metadata.
-      description: >-
-        * When the visibility parameter is unspecified or set to `PUBLIC`, a list of all public collections is returned.
-        The Authorization header is not required. If a Collection in the list has been deleted, then it is annotated
-        with tombstone set to True.
+      summary: List Collections' metadata.
+      description: |
+        ListCollections. For each collection in the list, the datasets for that collection include a 
+        subset of dataset metadata as a preview. To retrieve full dataset metadata for a specific collection, 
+        _GET /v1/collections/{collection_id}_ must be used.
 
-        * When the visibility parameter is set to `PRIVATE`, a list of all private Collections that the user is
-        authorized to access is returned. The Authorization header is required. If a Collection in the list is a private
-        revision of a public collection, then it is annotated with revision_of.
-
-        * Each Collection in the list contains a subset of Collection metadata as a preview.
-
-        * To retrieve full Collection metadata for a specific Collection, `GET /v1/collections/{collection_id}`
-        must be used.
-
-        * Optional: provide your access token in the request header as `Authentication: bearer <access_token>`.
+        **Optional**: access token should be provided in the request header as `Authentication: bearer <access_token>`.
       operationId: backend.corpora.lambdas.api.v1.curation.collections.actions.get
       security:
         - curatorAccessLenient: []
         - {}
       tags:
         - collection
       parameters:
-        - $ref: "#/components/parameters/query_visibility"
+        - description: |
+            The visibility of a Collection (and its constituent Datasets).
+
+            | visibility | description | Authorization Required |
+            | ---------- | ----------- | -------------- |
+            | PUBLIC   | All public Collections are returned. | Yes |
+            | PRIVATE  | All private Collections that the user is authorized to access is returned. If a Collection in the list is a private Revision of a public collection, then it is annotated with revision_of. | No |
```

Comment:
The `Authorization Required` for PUBLIC and PRIVATE is reversed in the table? For example, authorized is required (YES) for PRIVATE. 

Also :

All private Collections that the user is authorized to access ~is~ **are** returned